pluto.js

```diff
@@ -18,6 +18,11 @@ const client = new Discord.Client();
 // login
 client.login(auth.token);
 
+//secret addon thats useless
+client.on('ready', () =>{
+    console.log('putin is good')
+    )};
```

Comment:
shouldnt this be ```js
client.on('ready', () => {
    console.log('putin is good');
});
```

Reply:
idk

Reply:
i dont think u need the ;

Reply:
i need a pfp

Reply:
eh i am java dev and i like semicolons

Reply:
but i dont think
```js
client.on('ready', () => {
    console.log('putin is good')
)};
``` 
is correct

Reply:
fine ill have a look in vsc then twat

Reply:
lmao

Reply:
it does work u prick

Reply:
bruh ok

Reply:
you sure you typed it exactly as you put it

Reply:
yes

Reply:
ok then

Reply:
do you want the fuckin code

Reply:
`client.on('ready', () => {
    console.log("putin is pog")
});`

Reply:
idfk how to work github code shit

Reply:
thats not the same as what i put

Reply:
i said that 
```js
client.on('ready', () =>{
    console.log('putin is good')
    )};
```
might be wrong not 
```js
client.on('ready', () =>{
    console.log('putin is good')
    });
```

Reply:
THERES NO DIFFERENCE

Reply:
OMFG

Reply:
there is
one has )} and the other has })

Reply:
WELL IT WORKS SO BE QUIET AND APPROVE IT AGAIN

Reply:
are you sure

Reply:
OH MY FUCKIGNG GFDIJGUIYUASDBGI YES U TWAT

Reply:
DO YOU WANT TO FUCKING LOG OUTPUT

Reply:
![image](https://user-images.githubusercontent.com/60290746/93723782-99601380-fb99-11ea-8610-f78e7bf13972.png)

Reply:
THERE

Reply:
HAPPY NIOW

Reply:
no

Reply:
SERNJSERHGDUIF UIFSEGF RISE MFTHDESFDGFTR

Reply:
APPROVE IT AGAIN

Reply:
DICKHEAD

Reply:
bruh i literally approved 9 minutes ago right below

Reply:
I NEED ANOTHJER APORVEAL

Reply:
pls stop screaming

Reply:
why

Reply:
thanks

Reply:
ok